cmd/bnscli/clitests/send_tokens.test

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+keyfile=`tempfile`
+echo 00wZcK6QrPNAXy2Z3KyhbQx9s3n0vq/P32Z7nWnONQ0n9ftEBQnfp57Ig6BRC8mpYUw9RBiIgfDF5AKJi0vzyQ== | base64 -d > $keyfile
+
+bnscli send-tokens \
```

Comment:
Huh, keyfile isn't even used here. Did you forget the `sign` step?

Reply:
`sign` test was not possible because the connection to the server is required. I have implemented this for the tests now.